run_veracruz_server_test_tz.sh

```diff
@@ -9,7 +9,7 @@
 # and copyright information.
 
 # Run test individual phases:
-./run_tz_test.sh "veracruz_server_test test_phase1"  20
-./run_tz_test.sh "veracruz_server_test test_phase2"  20
-./run_tz_test.sh "veracruz_server_test test_phase3"  20
-./run_tz_test.sh "veracruz_server_test test_phase4"  40
+./run_tz_test.sh "veracruz_server_test test_phase1"  40
```

Comment:
Why was this wait value increased?

Reply:
In the CI, it seems to take slightly longer time, so I double the time here. I think probably few more seconds will work.